File: 1047.c
#include <stdio.h>
int main(){
    int h0, m0, hf, mf, horatotal, minutototal;
    scanf("%d %d %d %d",&h0, &m0, &hf, &mf);
    horatotal = hf - h0;
    minutototal = mf - m0;
    if(horatotal==0 && minutototal == 0){
        horatotal = 24;
    }
    if(mf<m0 && h0>hf){
        minutototal = 60 - (m0 - mf);
        horatotal = 24 + horatotal - 1;
    }
    else if(mf<m0 && h0<hf){
        minutototal = 60 - (m0 - mf);
        horatotal = horatotal - 1;
    }
    else if(mf<m0 && horatotal == 0) {
        minutototal = 60 - (m0 - mf);
        horatotal = 23;
    }
    printf("O JOGO DUROU %d HORA(S) E %d MINUTO(S)\n", horatotal, minutototal);

}

File: 1047.cs
using System;
class URI {
    static void Main(string[] args) { 
        String [] x = Console.ReadLine().Split(' ');
        int h0, m0, hf, mf, horatotal, minutototal;
        h0 = int.Parse(x[0]);
        m0 = int.Parse(x[1]);
        hf = int.Parse(x[2]);
        mf = int.Parse(x[3]);
        horatotal= hf - h0;
        minutototal = mf - m0;
        if(minutototal < 0){
            minutototal+= 60;
            horatotal -= 1;
        }
        if(horatotal < 0){
            horatotal += 24;
        }
        else if(h0 == hf && hf == mf && mf == m0){
            horatotal = 24;
            minutototal = 0;
        }
        Console.WriteLine($"O JOGO DUROU {horatotal} HORA(S) E {minutototal} MINUTO(S)");
    }
}
